packages/protobuf/README.md

```diff
@@ -0,0 +1,35 @@
+# `@penumbra-zone/protobuf`
+
+This package exports a `typeRegistry` (and inclusive `jsonOptions`) for use with
+`@bufbuild` and `@connectrpc` tooling, particularly
+`@penumbra-zone/transport-dom`.
+
+All message types necessary for a Connect `Transport` to serialize/deserialize
+communication with Prax or any other Penumbra extension are included.
+
+## If you simply need a Penumbra extension client
+
+You're looking for `@penumbra-zone/client`, which handles this process for you
+and also performs some basic safety checks.
+
+This package is provided for those who are interested in lower-level work or
+more detailed configuration.
+
+### Simple example
+
+```ts
+import { jsonOptions } from '@penumbra-zone/protobuf';
+
+import { createChannelTransport } from '@penumbra-zone/transport-dom';
+import type { ServiceType } from '@bufbuild/protobuf';
+
+// unsafely get first available provider
+const getPort = () => Object.values(window[Symbol.for('penumbra')])[0].connect();
```

Comment:
Shouldn't we be encouraging them to use `getAnyPenumbraPort` from the client package here instead?

Reply:
this example intentionally demonstrates use in isolation.

developers using the client package will use the features in the client package that conceal this, and will be redirected by the callout section before this example. anyone consuming this package instead will only be confused or distracted by example references to the client package.